FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -58,6 +60,7 @@
 static NSInteger gMaxRetryCount;
 static NSInteger gRetrySeconds;
 static bool gIsRetrying;
+static bool gIsRealtimeDisabled;
```

Comment:
Not sure where my comments go, but these values' read and write should be guarded by a serial queue to ensure accuracy.

Reply:
Do this in the patch PR plus realtimeDisabled flag only goes to true if activated, never back to false.